packages/opa/package.json

```diff
@@ -11,6 +11,7 @@
     "directory": "packages/opa"
   },
   "scripts": {
+    "test": "node --import tsx --test tests/**/*.ts",
```

Comment:
This change needs to be done via the proper setting in the gen.yaml, not like this. It's just for now, for testing the `@styra/opa` code in this PR.